src/main/scala/org/renci/cam/PredicatesService.scala

```diff
@@ -0,0 +1,37 @@
+package org.renci.cam
+
+import java.io.StringReader
+import java.nio.file.{Files, Paths}
+
+import com.typesafe.scalalogging.LazyLogging
+import io.circe.{Encoder, KeyEncoder}
+import io.circe.syntax._
+import org.apache.commons.csv.CSVFormat
+import org.renci.cam.Biolink.BiolinkData
+import org.renci.cam.HttpClient.HttpClient
+import org.renci.cam.domain.{BiolinkClass, BiolinkPredicate}
+import zio.config.ZConfig
+import zio.{Has, RIO, Task}
+
+import scala.jdk.CollectionConverters._
+
+object PredicatesService extends LazyLogging {
+
+  case class Triple(subj: BiolinkClass, pred: BiolinkPredicate, obj: BiolinkClass)
+
+  def run(): RIO[ZConfig[AppConfig] with HttpClient with Has[BiolinkData], String] =
```

Comment:
There is a general convention in Scala that methods with parentheses and no arguments execute side effects. This one does conceptually but since it returns a ZIO it's not actually executed. So I would just call this `def run: RIO...`